manifests/k/KDE/Kate/21.08.2/KDE.Kate.installer.yaml

```diff
@@ -1,14 +1,14 @@
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
-PackageIdentifier: KDE.Kate
-PackageVersion: 21.08.2
-MinimumOSVersion: 10.0.0.0
-InstallerType: nullsoft
-Scope: machine
+PackageIdentifier: KDE.Kate.Release
```

Comment:
In order to change a package identifier you have to change the folder that the manifest is inside. In this case, the entire manifest would need to be moved to `/manifests/k/KDE/Kate/Release/<version>`